src/main/java/com/marklogic/client/workspaces/WorkspacesExporter.java

```diff
@@ -0,0 +1,14 @@
+package com.marklogic.client.workspaces;
+
+import java.io.File;
+import java.util.List;
+import java.util.Set;
+
+import com.marklogic.client.DatabaseClient;
+
+public interface WorkspacesExporter {
+
+	Set<File> exportWorkspaces(String fileNameForWorkspace, String user,
+			String workspaceToDownload, DatabaseClient client);
```

Comment:
What does the "user" refer to here? 

I think this interface can be a bit simpler initially. Ultimate, I want to use this via a Gradle task - e.g. gradle mlExportWorkspaces would be the simplest thing. If I only want to export a single workspace, I'd run "gradle mlExportWorkspaces -PworkspaceName=test". Or possibly "gradle mlExportWorkspaces -PworkspaceName=test,test2" for when I want to select multiple workspaces.

I think we can default the name of each workspace file to "(name of workspace).xml". I think a workspace always has a name?

And I think we can default the location of workspaces to "./data/qconsole/workspaces". 

So I think the simplest interface would be:

    Set<File> exportWorkspaces(DatabaseClient client, String... workspaceNames)

Using "String..." allows for a user to specify no workspace names, in which case all workspaces are exported . I think that interface would suffice for most use cases.